src/modules/likes.js

```diff
@@ -1,31 +1,22 @@
-import { involvementApi, apiId } from './api.js';
+import { postLike, getLikes } from './api.js';
 
-// eslint-disable-next-line camelcase
-const addLike = async (item_id) => {
-  try {
-    const response = await fetch(`${involvementApi}/apps/${apiId}/likes`, {
-      method: 'POST',
-      body: JSON.stringify({
-        item_id,
-      }),
-      headers: {
-        Accept: 'application/json',
-        'Content-Type': 'application/json',
-      },
-    });
-    if (response.ok) {
-      return true; // Indicate success
-    }
-    const responseText = await response.text();
-    return response('Successfully created, like:', responseText);
-  } catch (error) {
-    return error('Error creating like:', error);
-  }
+const updateLikeCount = (likeCount, showId) => {
+  getLikes()
+    .then((likes) => {
+      const showLikes = likes.find((item) => item.item_id === showId);
+      likeCount.textContent = `${showLikes ? showLikes.likes : 0} Likes`;
+    })
+    .catch((error) => error);
 };
 
-const getLikes = async () => {
-  const response = await fetch(`${involvementApi}/apps/${apiId}/likes`);
-  return response.json();
-};
+function handleLikeButtonClick(likeBtn, likeCount) {
+  const showId = likeBtn.getAttribute('data-show-id');
```

Comment:
I would suggest adding one new line after line 13:
```
updateLikeCount(likeCount, showId);
```
I know that you are already using the `updateLikeCount` function on line 17 (don't delete it, you are absolutely right here) 💯 
But adding this function after line 13 will show the number of likes immediately when the page is loaded ⌛ 

Try it out and let me know if you like this approach ❓

Reply:
Oooh I will do that immediately